frontend/src/components/sidebar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import './sidebar.css'
+import Link from 'next/link'
+
+export default function Sidebar() {
+    return(
+        <div className='sidebar'>
+            <div className='logo'>
+                <img
+                    width={250} 
+                    height={100} 
+                    src="/UNLV.png"
+                    className='logo'
+                />
+            </div>
+            <Link href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" className='link'>Home</Link>
+            <Link href="/profile" className='link'>Explore</Link>
+            <Link href="/profile" className='link'>Hubs</Link>
+            <Link href="/profile" className='link'>Messages</Link>
```

Comment:
Messages, notifications, home, and profile do not have to be included as they will be included in the header navigation bar.

Reply:
After the above have been removed, then this can be merged.